contracts/rewards/src/state.rs

```diff
@@ -363,6 +394,30 @@ impl<T> Deref for StorageState<T> {
     }
 }
 
+pub(crate) fn load_rewards_pool(
+    storage: &dyn Storage,
+    pool_id: PoolId,
+) -> Result<RewardsPool, ContractError> {
+    POOLS
+        .may_load(storage, pool_id)
+        .change_context(ContractError::LoadRewardsPool)?
+        .ok_or(ContractError::RewardsPoolNotFound.into())
+}
+
+pub(crate) fn load_params(storage: &dyn Storage) -> StoredParams {
+    PARAMS.load(storage).expect("params should exist")
+}
+
+pub(crate) fn load_epoch_tally(
+    storage: &dyn Storage,
+    pool_id: PoolId,
+    epoch_num: u64,
+) -> Result<Option<EpochTally>, ContractError> {
+    TALLIES
+        .may_load(storage, TallyId { pool_id, epoch_num })
+        .change_context(ContractError::LoadEpochTally)
+}
+
```

Comment:
good first step to make it easier to delete the Store in a (separate) refactoring